### PR TITLE
Add link component

### PR DESCRIPTION
**Pull request by joaojusto**
Why:

* We need a way to render links;

This change addresses the need by:

* Adding a new component named `Link`
* Adding `<Link></Link>` to `kitchensink`;

Screenshot 2020-04-13 at 16 35 02

### CODE REVIEW

File: src/components/Link/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styles from "./index.module.css";
+
+const Link = ({ href, children }) => (
```
Comment (jfranciscosousa) [inline src/components/Link/index.js:7]: We need to distinguish external links from internal links `import Link from next/link`

Comment (joaojusto) [reply src/components/Link/index.js:7]: Nice catch, I'll change

Comment (joaojusto) [reply src/components/Link/index.js:7]: Updated

File: src/components/Button/index.js [hunk 2]
```diff
@@ -15,9 +16,10 @@ const Button = ({ theme, href, children, onClick }) => {
 
   if (href) {
     return (
-      <a className={className} href={href}>
-        {children}
-      </a>
+      <NextLink href={href}>
+        {/* eslint-disable-next-line */}
```
Comment (jfranciscosousa) [inline src/components/Button/index.js]: What are you disabling??? xD

Comment (joaojusto) [reply src/components/Button/index.js]: `a11y` - no href warning.

Comment (jfranciscosousa) [reply src/components/Button/index.js]: You can go to the `eslintrc` and change this rule: 
```
    "jsx-a11y/anchor-is-valid": [
      "error",
      {
        "components": [
          "Link"
        ],
        "specialLink": [
          "to"
        ]
      }
    ],
```

To include `NextLink` in that components list

Comment (joaojusto) [reply src/components/Button/index.js]: I'll change this to disable that specific rule

Comment (joaojusto) [reply src/components/Button/index.js]: Updated

**Review by jfranciscosousa — APPROVED** [review]

**Pull request merged**